packages/components/src/snackbar/types.ts

```diff
@@ -1,116 +1,42 @@
 /**
  * External dependencies
  */
-import type { MutableRefObject, ReactNode, SyntheticEvent } from 'react';
+import type { MutableRefObject, ReactNode } from 'react';
 
-export type NoticeActionWithURL = {
-	label: string;
-	url: string;
-	onClick?: ( event: SyntheticEvent ) => void;
-};
-
-type NoticeActionWithOnClick = {
-	label: string;
-	url?: string;
-	onClick: ( event: SyntheticEvent ) => void;
-};
-
-// TODO: move this type to the Notice component once it gets typed.
-export type NoticeAction = NoticeActionWithURL | NoticeActionWithOnClick;
-
-export type Notice = {
-	id: string;
-	spokenMessage: string;
-	actions: NoticeAction[];
-	icon?: ReactNode;
-	onDismiss?: () => void;
-	content: string;
-	isDismissible: boolean;
-	explicitDismiss: boolean;
-};
+/**
+ * Internal dependencies
+ */
+import type { NoticeProps, NoticeChildren } from '../notice/types';
 
-export type SnackbarProps = {
-	/**
-	 * The displayed message of a notice.
-	 *
-	 * Also used as the spoken message for assistive technology,
-	 * unless `spokenMessage` is provided as an alternative message.
-	 */
-	children: string;
-	/**
-	 * Used to provide a custom spoken message.
-	 *
-	 * @default children
-	 */
-	spokenMessage?: Notice[ 'spokenMessage' ];
-	/**
-	 * A politeness level for the notice's spoken message. Should be provided as
-	 * one of the valid options for an `aria-live` attribute value. Note that this
-	 * value should be considered a suggestion; assistive technologies may
-	 * override it based on internal heuristics.
-	 *
-	 * A value of `'assertive'` is to be used for important, and usually
-	 * time-sensitive, information. It will interrupt anything else the screen
-	 * reader is announcing in that moment.
-	 * A value of `'polite'` is to be used for advisory information. It should
-	 * not interrupt what the screen reader is announcing in that moment
-	 * (the "speech queue") or interrupt the current task.
-	 *
-	 * @see https://www.w3.org/TR/wai-aria-1.1/#aria-live
-	 *
-	 * @default 'polite'
-	 */
-	politeness?: 'polite' | 'assertive';
```

Comment:
The new definition of `politeness` in `Notice` is lacking a default value — should we override it for `Snackbar` if it makes sense?

Reply:
The default value to `politeness` in `Notice` is a bit tricky, as [it depends on the `status` prop's value](https://github.com/WordPress/gutenberg/blob/b875859cb5e31b20029d7519d492706af216a202/packages/components/src/notice/index.tsx#L41-L42). That's described in the JSDoc, but I've taken a pass at encapsulating it in an actual default declaration in 639be12afa. Let me know if you have suggestions on how to better format it!

Reply:
I see — sorry if I missed that! We can probably leave it as it is for now. The only alternative that comes to mind is to create a few type unions listing the different default values for `politeness` depending on `status` , i.e.

Reply:
Actually, just noticed (pun not intended) the changes that you recently pushed — that could probably work too! I'm going to mark this convo as resolved.

Reply:
aaaaand I **noticed** this comment, reverting the changes I'd pushed... then **noticed** your other comment after the fact and implemented the one-line version you suggested. So... I think we're on the same page/comment now? 😆